Updated setup docs and release notes

cjduffett opened the pull request.

Commit:
Updated setup docs and release notes

gregquinn2001 commented:
Need to add PostGIS and its version to software list. I guess we don't need to include tomcat or james/Direct RI yet

gregquinn2001 commented:
PostGIS is 2.x.
Also:
Apache Web Server
mod_proxy for apache web server

Commit:
Minor Revisions

Added missing software versions for Apache, PostGIS, and mod_proxy (could not find a version for mod_proxy)
